apps/web/components/Organization/Slice/MultipleStatistics/MultipleStatistics.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Link,
+  Text,
+} from '@island.is/island-ui/core'
+import { MultipleStatistics as MultipleStatisticsSchema } from '@island.is/web/graphql/schema'
+
+interface SliceProps {
+  slice: MultipleStatisticsSchema
+}
+
+export const MultipleStatistics: React.FC<SliceProps> = ({ slice }) => {
+  return (
+    <section key={slice.id} aria-labelledby={'sliceTitle-' + slice.id}>
+      <Box
+        borderTopWidth="standard"
+        borderColor="standard"
+        paddingTop={[4, 4, 6]}
+        paddingBottom={[4, 4, 6]}
+      >
+        {!!slice.title && (
+          <Text variant="h2" as="h2" marginBottom={4}>
+            {slice.title}
+          </Text>
+        )}
+        {slice.statistics.map((statistics) => (
+          <Box>
+            <Text as="h3" variant="h3" marginBottom={4}>
+              {statistics.title}
+            </Text>
+            <GridContainer>
+              <GridRow marginBottom={4}>
+                {statistics.statistics.map((s) => (
+                  <GridColumn span="1/5" paddingBottom={4}>
+                    <Text variant="h1" color="blue400">
```

Comment:
This ultimately renders as an `<h1>` tag.
Recently **Sjá** did an accessibility audit on Island.is and one of the things they mentioned were multiple H1 tags being unfavorable.

This might have to be taken into account since this is also violating the heading order for pages and therefore the question becomes whether steps should be taken for screen readers.

Reply:
It shows up as `<p>` because I don't have `as="h1` :)

![image](https://user-images.githubusercontent.com/3607456/121051979-75854200-c7a9-11eb-8cb1-ee86758bdd61.png)

Reply:
Oh whoops, my mistake :+1: